### PR TITLE
intial commit

### PR DESCRIPTION
**Pull request by pratyusha3022**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,2 @@
 # Pluralsight Spring Boot Application Resources
-## Author: Dan Bunker
```
Comment (arunece488) [inline README.md:2]: y this chNge

**Review by arunece488 — CHANGES_REQUESTED** [review]
looks good

File: README.md (hunk 1 above)
Comment (arunece488) [reply README.md:2]: @pratyusha3022  y this change

Comment (pratyusha3022) [reply README.md:2]: needed this

**Review by arunece488 — APPROVED** [review]

**Review by arunece488 — APPROVED** [review]

**Pull request merged**